### PR TITLE
Add anchors in the documentation

### PR DESCRIPTION
**Pull request by colinepiloquet**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No

**What kind of change does this PR introduce?**

Docs update

**What is the current behavior?**

A lot of sections of the documentation have no anchors so they cannot be referenced to in the documentation of other repositories.

**What is the new behavior (if this is a feature change)?**
Add some anchors

### CODE REVIEW

**Comment by colinepiloquet** [comment]
@So-Fras, there might be some missing but it's a start and it covers what I need for the tutorials for now (actually, way more). Maybe I did not choose the best names sometimes though.

File: docs/grid_features/loadflow_validation.md [hunk 1]
```diff
@@ -89,9 +93,11 @@ V - targetV & < & -& \epsilon && \& && |Q-maxQ| & \leq & \epsilon \\
 targetV - V & < && \epsilon && \& && |Q-minQ| & \leq & \epsilon \\
 \end{align*}
 
+(loadflow-validation-laods)=
```
Comment (So-Fras) [inline docs/grid_features/loadflow_validation.md:96]:
```suggestion
(loadflow-validation-loads)=
```

**Review by So-Fras — CHANGES_REQUESTED** [review]
Thanks for your work!

File: docs/grid_features/loadflow_validation.md (hunk 1 above)
Comment (colinepiloquet) [reply docs/grid_features/loadflow_validation.md:96]: Oups

**Review by So-Fras — APPROVED** [review]

**Pull request merged**